src/main/java/io/neonbee/endpoint/metrics/NeonBeePrometheusHandler.java

```diff
@@ -0,0 +1,63 @@
+package io.neonbee.endpoint.metrics;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import io.neonbee.logging.LoggingFacade;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.prometheus.client.exporter.common.TextFormat;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.micrometer.backends.BackendRegistries;
+import io.vertx.micrometer.impl.PrometheusScrapingHandlerImpl;
+
+/**
+ * A Vert.x Web {@link io.vertx.ext.web.Route} handler for Prometheus metrics scraping.
+ * <p>
+ * The original Implementation doesn't work with {@link CompositeMeterRegistry}. This implementation fixes this.
+ */
+public class NeonBeePrometheusHandler extends PrometheusScrapingHandlerImpl {
```

Comment:
I think we shall keep the original name. Call it:

PrometheusScrapingHandler (as it is in our NeonBee package, it is clear that it is ours).

Alternatively (please maybe quickly discuss / share with Pascal / Sebastian) call it:

NeonBeePrometheusScrapingHandler